packages/dev-server-core/src/server/DevServer.ts

```diff
@@ -19,6 +19,7 @@ export class DevServer {
     public config: DevServerCoreConfig,
     public logger: Logger,
     public fileWatcher = chokidar.watch([]),
+    externalServer?: Server,
```

Comment:
another question on the same line: is that public/private? looks like it should always explicit in this file

Reply:
What should be explicit? My understanding is the use of `public/private` in the constructor triggers some syntax sugar that automatically sets these values to `this.config = config`, etc. However, `externalServer` isn't needed beyond the call to `new WebSocketsManager(externalServer)`, as far as I can understand.

Reply:
my bad, I thought it was used outside of the constructor, but it's not
so code is good